packages/plugin-ext/src/api/rpc-protocol.ts

```diff
@@ -357,7 +358,7 @@ class MessageFactory {
 namespace ObjectsTransferrer {
 
     // tslint:disable-next-line:no-any
-    export function replacer(key: string | undefined, value: any): any {
+    export function replacer(cache: any[], key: string | undefined, value: any): any {
```

Comment:
Out of curiosity; why no `export function replacer(key: string | undefined, value: any, cache: any[] = []): any {`?

Reply:
done